GG/src/RichText/ImageBlock.cpp

```diff
@@ -15,13 +15,14 @@ namespace GG {
     const std::string ImageBlock::IMAGE_TAG("img");
 
     ImageBlock::ImageBlock(const std::string& image_path, X x, Y y, X w,
-                           GG::Flags<GG::WndFlag> flags)
-    : BlockControl(x, y, w, flags), m_graphic(0)
+                           GG::Flags<GG::WndFlag> flags):
+        BlockControl(x, y, w, flags), m_graphic(0)
     {
         try {
             boost::shared_ptr<Texture> texture = GetTextureManager().GetTexture(image_path);
-            m_graphic =
-                new StaticGraphic(texture, GRAPHIC_PROPSCALE | GRAPHIC_SHRINKFIT | GRAPHIC_CENTER);
+            m_graphic = new StaticGraphic(
```

Comment:
why all the linebreaks...? could just move the new ... to the same line as m_graphic =